src/VisualStudio/CSharp/Impl/Options/AutomationObject/AutomationObject.Style.cs

```diff
@@ -244,12 +244,6 @@ public string Style_PreferIsNullCheckOverReferenceEqualityMethod
             set { SetXmlOption(CodeStyleOptions2.PreferIsNullCheckOverReferenceEqualityMethod, value); }
         }
 
-        public string Style_PreferParameterNullChecking
-        {
-            get { return GetXmlOption(CSharpCodeStyleOptions.PreferParameterNullChecking); }
-            set { SetXmlOption(CSharpCodeStyleOptions.PreferParameterNullChecking, value); }
-        }
```

Comment:
Does this change hurt backward compatibility?

https://github.com/dotnet/roslyn/blob/3fcc9af35de964c195ff764b32e360d53e4cb5ac/src/VisualStudio/CSharp/Impl/Options/AutomationObject/AutomationObject.ObsoleteAndUnused.cs#L12-L18

If not we could delete `AutomationObject.ObsoleteAndUnused.cs`. I think a good way of testing that is:

1. Export VS settings from the current version which has the option.
2. Make sure the produced file contains this option
3. Try to load the file from a VS build from this PR that doesn't have an option.
4. Will we try access the getter that doesn't exist?

Reply:
I'll try this when I get the chance. Thanks.